K8s-deployment/Charts/geo-server/install.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+kubectl create namespace ugix-geoserver
```

Comment:
do no not give name ugix in namespace ugix-geoserver, just geoserver is fine

Reply:
okay

Reply:
suggested changes have been updated ! please check